components/debounce/browser/debounce_component_installer.cc

```diff
@@ -67,7 +67,7 @@ void DebounceComponentInstaller::OnDATFileDataReady(
             net::registry_controlled_domains::GetDomainAndRegistry(
                 pattern.host(),
                 net::registry_controlled_domains::PrivateRegistryFilter::
-                    INCLUDE_PRIVATE_REGISTRIES);
+                    EXCLUDE_PRIVATE_REGISTRIES);
```

Comment:
can you create a debounce utility method so we don't have to keep these two calls in sync? Since we're looking at a follow-up anyway, I'm also ok with a comment that we have to keep them in sync for now